### ISSUE TITLE
[FEA] For the spark-rapids CI, add equivalent labels for [skip ci] and [databricks] 

### ISSUE THREAD

**Issue by razajafri** [NVIDIA/spark-rapids#5928]
**Is your feature request related to a problem? Please describe.**
It's very easy to misspell the tags in the description. Even case changes i.e. databricks and Databricks have to be accounted for in the script

**Describe the solution you'd like**
Is it possible to have labels that can be read by the script instead?

**Comment by pxLi** [comment]
Yeah, we could add this feature.

how should we resolve conflicts between title suffix and label, e.g.
title contains [skip ci], but label have `databricks`, should we trigger databricks CI or simply skip ci?
If title only, `skip ci` always win against other phrases

We may require some priority for title + label cases, any suggestion? @sameerz @razajafri

**Comment by sameerz** [comment]
@pxLi can we replace the title trigger with labels triggers only?

**Comment by pxLi** [comment]
> @pxLi can we replace the title trigger with labels triggers only?

yes, we could. But better to have an interim period. So we could keep both but label have higher priority during this period and then we remove the title support

**Comment by razajafri** [comment]
IMO `skip ci` should take precedence over `databricks`.

**Comment by jlowe** [comment]
Here's my $0.02:
- There should be no "priority" between title tags and labels, they are effectively equivalent in every way.  Directives from title tags and labels should be acted on regardless of where they originated.
- The skip CI directive skips all CI regardless of other directives.  Bonus points if the skipping is as visible as possible in the Github PR checks summary.

**Comment by pxLi** [comment]
I will start working on this, we will have our own shared jenkins github lib shortly instead of relying blossom-provided one.
It would be a good time we move https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/jenkins/Jenkinsfile-blossom.premerge back to internal gitlab repo for better development, maintenance and previous security concern
